### PR TITLE
[Android] Disable click on NFTs; filter accounts for NFTs

### PR DESCRIPTION
**Pull request by wchen342**
Resolves brave/brave-browser/issues/23507

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [x] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:

1. Go to wallet and add an NFT.
2. The NFT shall be non-clickable.

### CODE REVIEW

File: android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java [hunk 1]
```diff
@@ -302,6 +302,9 @@ public void onAssetClick(BlockchainToken asset) {
         if (selectedNetwork == null) {
             return;
         }
+        // TODO (Wengling): re-enable when NFT portfolio is done
+        if (asset.isNft) return;
```
[the remark this answers is not in the document]
Comment (SergeyZhukovsky) [reply android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java:306]: yeah, but when portfolio is done fetching through all accounts, does it allow to click on it?

[the remark this answers is not in the document]
Comment (wchen342) [reply android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java:306]: Isn't NFT portfolio a new page (not on Android yet) to show the content of the NFT, like on Desktop? I think he didn't meant the current portfolio page for wallet.

[the remark this answers is not in the document]
Comment (wchen342) [reply android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java:306]: Let me check with him and get back.

[the remark this answers is not in the document]
Comment (wchen342) [reply android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java:306]: @SergeyZhukovsky My understanding was correct, he was talking about the desktop NFT details page which doesn't exist on Android yet, so I think it is reasonable we disable clicking until it is implemented.

[the remark this answers is not in the document]
Comment (SergeyZhukovsky) [reply android/java/org/chromium/chrome/browser/crypto_wallet/fragments/PortfolioFragment.java:306]: ok, sounds good!

**Review by SergeyZhukovsky — APPROVED** [review]
++

**Pull request merged**